modin/_compat/core/pandas_common.py

```diff
@@ -16,20 +16,43 @@
 from modin._compat import PandasCompatVersion
 
 if PandasCompatVersion.CURRENT == PandasCompatVersion.PY36:
-    from .py36.pd_common import (
+    from .py36.pandas_common import (
         get_handle,
-        pd_pivot_table,
-        pd_convert_dtypes,
+        pandas_pivot_table,
+        pandas_convert_dtypes,
+        pd_compare,
+        pandas_dataframe_join,
         reconstruct_func,
+        pandas_reset_index,
+        pandas_to_csv,
+        DataError,
+        SpecificationError,
     )
 
 
 elif PandasCompatVersion.CURRENT == PandasCompatVersion.LATEST:
-    from .latest.pd_common import (
+    from .latest.pandas_common import (
         get_handle,
-        pd_pivot_table,
-        pd_convert_dtypes,
+        pandas_pivot_table,
+        pandas_convert_dtypes,
+        pd_compare,
```

Comment:
```suggestion
        pandas_compare,
```

Reply:
@vnlitvinov perhaps this message was lost among many others :)

Reply:
@vnlitvinov, ping

Reply:
fixed